app/assets/v2/js/pages/profile.js

```diff
@@ -10,8 +10,30 @@ $(document).ready(function() {
 
   setupTabs('#activity-tabs');
 
+  let file = '';
+
+  $('body').on('click', 'img.picky', function() {
```

Comment:
hmmm i wonder if we should factor out profile info, from profile-edit into a profile-edit.js.  @danlipert might have a preference here.

Reply:
@danlipert what do you think?

Reply:
@owocki @iamonuwa yeah I think factoring it out into its own file is a good idea